docs/src/hal/comp.adoc

```diff
@@ -272,15 +272,25 @@ parameters, and functions for `prefix`.
   Its return type is 'void'; it may call 'exit()' if it wishes to terminate rather than create a HAL component
   (e.g., because the commandline arguments were invalid).
 
-* 'option extra_link_args "..."' - (default: "") +
-  This option is ignored if the option 'userspace' (see above) is set to 'no'.
-  When linking a userspace component, the arguments given are inserted in the link line.
-  Note that because compilation takes place in a temporary directory,
-  "-L." refers to the temporary directory and not the directory where the .comp source file resides.
-
-* 'option extra_compile_args "..."' - (default: "") +
-  This option is ignored if the option 'userspace' (see above) is set to 'no'.
-  When compiling a userspace component, the arguments given are inserted in the compiler command line.
+* 'option extra_link_args "..."' - (default: "")
+   This option is ignored if the option 'userspace' (see above) is set to
```

Comment:
I had seen the "semantic breaks" phrase without actually thinking about what it meant. 
Basically lines should wrap at punctuation?

Reply:
Yes, that is what it most often boils down to. And I have even introduced the one or other comma when I had felt that it would make a sentence easier to understand :)
To explain the problem a bit better, those translators aim at mimicking the overall looks of a text. And newlines are typically reserved for paragraphs in word (where they hence have a meaning). The translators then give their best to have something grammatical before and after the line break - which is not always helpful.
A nice side-effect is that the diffs in git become easier to read.